### ISSUE TITLE
dns_server role enhancement: add ability to set IP addresses for external domains

### ISSUE THREAD

**Issue by neilmunday** [bluebanquise/bluebanquise#608]
Scenario:

Cluster with no Internet access.
Many BMCs set to use public NTP servers - too many to change by hand/not possible.
= lots of DNS errors in the logs

Is it possible for the dns_role to have the ability to return an IP for a given host?

E.g. in the example return the cluster's chrony server IP for pool.ntp.org?

@oxedions asked me to tag @btravouillon ;-)

**Comment by neilmunday** [comment]
See pull request #609 for a potential fix.

**Comment by btravouillon** [comment]
Seems a bit hackish but works, thanks for sharing.
That being said, you should have a serious talk with your hardware manufacturer if they don't provide a simple way to update BMC configuration at scale.

**Comment by neilmunday** [comment]
Agreed it is hackish and I've already raised the issue with the hardware manufacturer. You never know, the feature may be useful other scenarios in the future.

**Comment by btravouillon** [comment]
No doubt this is a useful feature. Thanks Neil!

**Comment by oxedions** [comment]
PR is merged, thanks a lot for this feature !
